index.html

```diff
@@ -29,64 +29,73 @@
+        </div>
+    </section>
+    <section id="faq">
+        <div class="container-fluid">
+            <div class="faq">
+                <h3>FAQs</h3>
+                <div class="row">
+                    <div class="col-lg-4">
+                        <div class="faq-block">
+                            <h4 class="faq-title">What is HAX?</h4>
+                            <p class="faq-text">HAX is an up and coming organization dedicated to helping people organize and hold hackathons, with a specific focus on women and disadvantaged minorities. We aim to provide all women and minorities a chance to receive and hands-on experience with programming and computer science, through bringing hackathons to high schools and overlooked areas all over the world!</p>
+                        </div>
                     </div>
-            </div>
-            <div class="col-lg-4">
-                    <div class="col-lg-12">
-                        <h4>Who started HAX?</h4>
-                        <p>HAX is the brain-child of Justina Chua and a team of absolutely charming, dilligent high school and college students. Hours have been put into making this organization a success. Sound interesting? Help us achieve our mission!</p>
+                    <div class="col-lg-4">
+                        <div class="faq-block">
+                            <h4 class="faq-title">How can I join?</h4>
+                            <p class="faq-text">We have 7 separate teams: Leadership, Content Creation, Outreach, Social Media, Management, Design, and Global Teams. You can apply to join multiple teams. Many potential team members expressed interest in getting involved in more than one team, so if you are not a member of the Core or Ambassador Team, you can be involved in a maximum of 3 teams. Make sure not to overexert yourself; while we are as passionate about this as you are, we do not expect this to take priority over school work! To apply, follow <a href="https://bit.ly/HAXapply" target="_">this link</a>.</p>
+                        </div>
                     </div>
+                    <div class="col-lg-4">
+                        <div class="faq-block">
+                            <h4 class="faq-title">Who started HAX?</h4>
+                            <p class="faq-text">HAX is the brain-child of Justina Chua and a team of absolutely charming, dilligent high school and college students. Hours have been put into making this organization a success. Sound interesting? Help us achieve our mission!</p>
+                        </div>
+                    </div>
+                </div>
             </div>
         </div>
-    </div>
+    </section>
+</div>
 </body>
 <footer>
     <div id="footer">
         <br></br>
         <a href="mailto:team@haxorg.io" style="text-decoration:none;">
-            <i class="fas fa-envelope" style="font-size: 24px; color: #6800fa; margin: 4px;"></i>

```

Comment:
I assume the padding and color has been put into the CSS file?

Reply:
Yep.